billing/lib/ucan-stream.js

```diff
@@ -6,16 +6,23 @@ import * as StoreCaps from '@web3-storage/capabilities/store'
  * the delta.
  *
  * @param {import('./api.js').UcanStreamMessage[]} messages
- * @returns {import('./api.js').UsageDelta[]}
+ * @param {{ storeTable: Pick<import('@web3-storage/upload-api').StoreTable, 'exists'> }} ctx
+ * @returns {Promise<import('./api.js').UsageDelta[]>}
  */
-export const findSpaceUsageDeltas = messages => {
+export const findSpaceUsageDeltas = async (messages, ctx) => {
   const deltas = []
   for (const message of messages) {
     if (!isReceipt(message)) continue
 
     /** @type {number|undefined} */
     let size
     if (isReceiptForCapability(message, StoreCaps.add) && isStoreAddSuccess(message.out)) {
+      // If status is done, we need to check if the item is stored in _this_
+      // space or not. If it is, then there is no delta.
+      if (message.out.ok.status === 'done') {
```

Comment:
do we increment when the space is allocated by calling `store/add`? If I keep calling `store/add` but not uploading the thing do i keep getting charged additional increments each time?